src/components/fields/SchemaField.js

```diff
@@ -214,6 +217,7 @@ if (process.env.NODE_ENV !== "production") {
       fields: PropTypes.objectOf(PropTypes.func).isRequired,
       definitions: PropTypes.object.isRequired,
       FieldTemplate: PropTypes.func,
+      formContext: PropTypes.object.isRequired
```

Comment:
Please leave trailing commas as this is the general style for this project (here and across the patch).